src/Modal/index.jsx

```diff
@@ -23,6 +23,8 @@ class Modal extends React.Component {
     this.el = document.createElement('div');
 
     this.state = {
+      // Sets true for IE11, false otherwise: https://stackoverflow.com/a/22082397/6620612
+      isIE11: !!window.MSInputMethodContext && !!document.documentMode,
```

Comment:
Why should this be part of the `state`? This doesn't really change through the lifetime of the component right?

Reply:
I'll make it an instance variable instead. I needed it to be set on component construction instead of import because of the test.